pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -182,3 +135,102 @@ func getDefaultInstrumentation(agentConfig *adapters.CwaConfig, isWindowsPod boo
 		},
 	}, nil
 }
+
+func getJavaEnvs(isAppSignalsEnabled bool, cloudwatchAgentServiceEndpoint, exporterPrefix string, additionalEnvs map[string]string) []corev1.EnvVar {
+	envs := []corev1.EnvVar{
+		{Name: "OTEL_EXPORTER_OTLP_PROTOCOL", Value: "http/protobuf"},
+		{Name: "OTEL_METRICS_EXPORTER", Value: "none"},
+		{Name: "OTEL_LOGS_EXPORTER", Value: "none"},
+	}
+
+	if isAppSignalsEnabled {
```

Comment:
If AppSignals isn't set, then it might make sense to set
```
{Name: "OTEL_TRACES_EXPORTER", Value: "none"},
```
https://opentelemetry.io/docs/languages/sdk-configuration/general/#otel_traces_exporter

With it enabled and AppSignals disabled, the SDK will log those `Failed to connect` errors, which can pollute customer application logs.